play-services-core/src/main/kotlin/org/microg/gms/auth/signin/AuthSignInActivity.kt

```diff
@@ -53,7 +53,7 @@ class AuthSignInActivity : AppCompatActivity() {
             intent?.extras?.also { it.classLoader = SignInConfiguration::class.java.classLoader }?.getParcelable<SignInConfiguration>("config")
         }.getOrNull()
 
-    private val Int.px: Int get() = (this * resources.displayMetrics.density).toInt()
+    private lateinit var accountManager: AccountManager
```

Comment:
It would be better to not keep this reference but instead just re-invoke `getSystemService<AccountManager>()` in `initView()`.